word_nerd/flashcards/views.py

```diff
@@ -35,3 +35,12 @@ def post(self, request, pk):
 
         return Response({'detail': 'Language is already selected.'},
                         status=status.HTTP_409_CONFLICT)
+
+
+class GetUserLanguagesAPIView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        languages = request.user.languages.all()
+        serializer = LanguageSerializer(languages, many=True)
+        return Response({'languages': serializer.data})
```

Comment:
I suggest making the result here and in `AllLanguagesAPIView` consistent, i.e. here you have `return Response({'languages': serializer.data})` and the other view has `return Response(serializer.data)`. Also, do we need `order_by` here as well?